web/src/views/HomepageView.jsx

```diff
@@ -1,54 +1,69 @@
 import React, { useState } from "react";
-import { Carousel } from "antd";
+import { Carousel, Row, Column } from "antd";
 import { Divider, Image } from "antd";
 
 const contentStyle = {
```

Comment:
I've read that, it's not a good approach to include inline CSS in the jsx file. Can you move them to another CSS file ?  You can refer to [here](https://www.sitepoint.com/react-components-styling-options/) for React component styling options.